xml/OptionPanel.xml

```diff
@@ -1,108 +1,182 @@
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Token Arranger"</Text>
+                <Text class="description">See the contents of the chaos bag at a glance! This tool displays a sorted table of the tokens to allow easier guessing of your odds.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showTokenArranger" onValueChanged="onClick_toggleOption(showTokenArranger)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show clean up helper -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Clean Up Helper"</Text>
+                <Text class="description">Useful for campaign-play: It resets play areas to allow continuous gameplay in the same savegame.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showCleanUpHelper" onValueChanged="onClick_toggleOption(showCleanUpHelper)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show hand helper -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Hand Helper"</Text>
+                <Text class="description">Never count your hand cards again! This tool does that for you and can even take "Dream-Enhancing Serum" into account. Also includes a button for randomly discard a card.</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showHandHelper" onValueChanged="onClick_toggleOption(showHandHelper)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show chaos bag manager -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Chaos Bag Manager"</Text>
+                <Text class="description">Panel for easy addition or removal of chaos tokens to the bag - very useful for EotE because of Frost tokens!</Text>
+              </VerticalLayout>
+            </Cell>
+            <Cell class="option-button">
+              <Toggle id="showChaosBagManager" onValueChanged="onClick_toggleOption(showChaosBagManager)"/>
+            </Cell>
+          </Row>
+
+          <!-- Option: show navigation overlay -->
+          <Row class="option-text">
+            <Cell class="option-text">
+              <VerticalLayout class="text-column">
+                <Text class="option-header">Show "Navigation Overlay"</Text>
+                <Text class="description">This enables an overlay for quickly navigating to various points on the table.</Text>
```

Comment:
Maybe the description here should explicitly specify that it's moving the camera around?

Reply:
Does "navigating to points on the table" not imply camera movement?

Reply:
Maybe?  "Navigate" feels like an overloaded word, and a bit ambiguous about what is being navigated.  It also doesn't match the core TTS terminology of camera save states.

Reply:
Leaving this one up to you as well, and resolving